src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -38,31 +54,79 @@ const visibleColumns = [
 
 // TODO: remove formatter for jobProfileName once backend is in place
 const JobLogsContainer = props => {
-  const { intl } = props;
+  const {
+    intl,
+    stripes: { okapi },
+  } = props;
 
   const {
     logs,
     hasLoaded,
   } = useContext(DataFetcherContext);
+  const calloutRef = useRef(null);
+
+  const handleDownloadError = () => {
+    if (!calloutRef.current) return;
+
+    calloutRef.current.sendCallout({
+      type: 'error',
+      message: <FormattedMessage id="ui-data-export.communicationProblem" />,
+    });
+  };
+
+  const downloadExportFile = async record => {
+    try {
+      const fileName = get(record.exportedFiles, '0.fileName');
+      const downloadLink = await getFileDownloadLink(record, okapi);
+
+      await downloadFileByLink(fileName, downloadLink);
+    } catch (error) {
+      handleDownloadError();
+
+      console.error(error); // eslint-disable-line no-console
+    }
+  };
+
+  const getFileNameField = record => {
+    const successRec = record.status === JOB_EXECUTION_STATUSES.SUCCESS;
```

Comment:
```suggestion
    const fileName = get(record.exportedFiles, '0.fileName');
    const isSuccessfulRecord = record.status === JOB_EXECUTION_STATUSES.SUCCESS;
    
    if (!isSuccessfulRecord) return fileName;
    
    return (
      <Button
        data-test-download-file-btn
        buttonStyle="link"
        marginBottom0
        onClick={() => downloadExportFile(record)}
      >
        {fileName}
      </Button>
    );
    
```

Reply:
It looks better, but with such approach I have this styles:
![Screenshot 2020-03-13 at 17 01 06](https://user-images.githubusercontent.com/60929399/76632577-17d55c00-654c-11ea-8a29-50784db06471.png)

Is it okay?

Reply:
Good catch, I think it makes sense to wrap the fileName in b.